docs/robots.md

```diff
@@ -0,0 +1,77 @@
+# Robots
+
+## Robots API
+
+Altis v7 introduced a filter-based Robots API, providing central control over the `robots` meta tag. The `robots` meta tag allows you to utilize a more granular, page-specific approach to controlling how an individual page should be indexed and served to users in search engine results. The meta tag is usually placed in the `<head>` section of a page.
```

Comment:
I don't think it's necessary to mention the version of altis here, we don't really do that anywhere in the docs other than the upgrade guide. Given we're moving to the docs site version with docs for all versions this shouldn't be an issue.
```suggestion
The Robots API provides central control over the `robots` meta tag. The `robots` meta tag allows you to utilize a more granular, page-specific approach to controlling how an individual page should be indexed and served to users in search engine results. The meta tag is automatically placed in the `<head>` section of a page.
```